### DISCUSSION TITLE
Unable to connet through Azure Application Gateway to EMQX websocket

### DISCUSSION THREAD

**Discussion by jian1117** [emqx/emqx#12654] [Q&A]
I deployed a emqx5.1.3 container on a Azure VM as a websocket backend pool (e.g., http://10.0.0.1:8083), and use Azure Application Gateway (e.g., https://mqtt.test.com:8084) in front of it. But I have issues below, any suggestion? Thanks.

- Unable to connet the URL (wss://mqtt.test.com:8084) by using MQTTX. 
- wscat -c wss://mqtt.test.com:8084/mqtt
error: Unexpected server response: 400
- wscat -c wss://mqtt.test.com:8084/mqtt
error: Unexpected server response: 404

**Comment by zmstone** [comment]
@jian1117 please enable debug level log and share the logs with us.
